portal-api/src/v1/controllers/deal.controller.js

```diff
@@ -75,7 +75,11 @@ const createNewDealData = async (deal, maker) => {
  * Create a deal (BSS, EWCS only)
  */
 const createDeal = async (dealBody, user) => {
-  const deal = await createNewDealData(dealBody, user);
+  const dealData = {
```

Comment:
For now I've moved the sanitization to the portal UI instead of in API. Not sure if in long term we also want anything to exist within portal API essentially replicating the current work or if that is harder to maintain. Portal UI has a better idea of what information to send so would be keen to keep the logic there and then later down the line move to using typescript and any package we can use for some schemas for the entities we save in the noSQL database